src/components/PersonTabs/PeopleTabs.tsx

```diff
@@ -0,0 +1,121 @@
+import { Loader } from '../Loader';
+import { useEffect, useState } from 'react';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
```

Comment:
The import path '././api' seems incorrect. It should be './api' to correctly resolve the module path.